pyproject.toml

```diff
@@ -43,6 +48,5 @@ ignore = [
 
 [tool.pytest.ini_options]
 minversion = "8.3.3"
-addopts = "--cov=src"
```

Comment:
I had to remove this as when this option is set by default the debugger does not work properly.

Reply:
> Option should be added to Makefile and .github/workflows/pytest.yml instead. pytest.yml is using wrongly modules instead of src.

I already fixed the call in the _pytest.yml_. But what would I need to add to the makefile?

Reply:
I would use the same options as in pytest.yml:

```makefile
# Target to run tests.
test:
	@echo "Running tests..."
        .venv/bin/pytest -vs --cov src --cov-report term-missing
```

Reply:
Done2